lib/server.js

```diff
@@ -763,7 +763,7 @@ class WebThingServer {
         });
       this.mdns.start();
       this.server.listen(this.port);
-    });
+    }).catch(console.error);
```

Comment:
This encapsulates any errors in starting the server. A script has no way of telling whether the start was successful.
The scope of this library depends on what to do here.

A failure could cause it
* to retry after a delay, (how would the executing script know?)
* to print an error to the console and exit the process with an error code. (might be too drastic if this is not the only purpose of the script importing this library)
* or to forward the promise to the script that executes `start()`. (my favourite, however, it’s a breaking change as every script will have to handle the promise)